src/pointings_search/pointing_table.py

```diff
@@ -332,4 +346,46 @@ def to_csv(self, filename, overwrite=False):
         overwrite: `bool`
             Whether to overwrite an existing file.
         """
-        self.pointings.write(filename, overwrite=overwrite)
+        self.pointings.write(filename, overwrite=False)
+
+    def to_sqlite(self, db_name, table_name, overwrite=False):
+        """Write the SQL data to a sqlite database potentially
+        overwriting an existing table.
+
+        Parameters
+        ----------
+        db_name : `str`
+            The file location of pointing database.
+        table_name : `str`
+            The table to query from the database.
+        overwrite : `bool`
+            A Boolean indicating whether to overwrite the an existing table.
+        """
+        # Check if the table exists.
+        connection = sqlite3.connect(db_name)
+        test_cursor = connection.cursor()
+        try:
+            reader = test_cursor.execute(f"SELECT * FROM {table_name} LIMIT 1")
+            if reader.fetchone() is not None:
+                if not overwrite:
+                    raise ValueError(f"Table {table_name} exists.")
+                else:
+                    cursor.execute(f"DROP TABLE {table_name}")
+        except sqlite3.OperationalError:
+            pass
+
+        # Create the table.
+        column_names = ", ".join(self.pointings.columns)
+        connection.execute(f"CREATE TABLE {table_name} ({column_names})")
+        print(f"CREATE TABLE {table_name} ({column_names})")
+
+        # Insert the rows.
+        write_cursor = connection.cursor()
+        for i in range(len(self.pointings)):
```

Comment:
Does astropy Table not let you do `for pointing in self.pointings:`?

Reply:
Fixed.